core/src/main/scripts/importer/validateData.py

```diff
@@ -3086,6 +3079,9 @@ def validateExonsInTranscript(self, transcript, exon, line_number):
             # For every fusion event, check if exon is part of the transcript
             validateExons(self)
 
+            # List all transcripts
```

Comment:
was this moved because there are no transcripts if self.transcirpt_set is <= 0? Haven't looked at the function in detail or where self.transcript_set is set.

Reply:
@averyniceday correct